src/openapi.yaml

```diff
@@ -254,6 +254,8 @@ paths:
     $ref: './resources/OrganizationBillingInfo.yaml'
   /organization/{organizationId}/billingStatus:
     $ref: './resources/OrganizationBillingStatus.yaml'
+  /organization/{organizationId}/billingExternalId:
+    $ref: './resources/OrganizationBillingStatus.yaml'
```

Comment:
should be OrganizationBillingExternalId I guess

Reply:
thank you @pggb25 it's fixed